src/topp/DecoyDatabase.cpp

```diff
@@ -54,6 +61,11 @@ and terminates the program if decoys are found.
 @verbinclude TOPP_DecoyDatabase.cli
 <B>INI file documentation of this tool:</B>
 @htmlinclude TOPP_DecoyDatabase.html
+
+
+The Neighbor Peptide functionality in the TOPPDecoyDatabase class is designed to find peptides from a given set of sequences (FASTA file) that are
+similar to a target peptide based on mass and spectral characteristics. This section will detail the methods and functionalities specifically related
+to the Neighbor Peptide search.
```

Comment:
extend on this, or at least reference the NeighborSeq class here, which has more details